schemas/events.v1.schema.json

```diff
@@ -1087,7 +1087,8 @@
         "expectstaple",
         "transaction",
         "nel",
-        "default"
+        "default",
+        "feedback"
```

Comment:
Just out of curiosity, how come we're changing `events` too? Is it not just an issue platform change?

Reply:
feedbacks are going to be a new event type in addition to new issue type.